Android sdk

panchy opened the pull request.

joepadmiraal commented on docs/index.html:
```diff
@@ -17,20 +17,21 @@
-        /* Change the theme color hue */
-        --theme-hue: 300;
-      }
+        :root {
```
Why did the indentation change here?
Seems like the rest of the file is 2 spaces instead of 4.

difosfor replied:
Looks like this is due to incorrect css within the style block; there is a superfluous closing bracket ("}") apparently resulting in prettier increasing indentation here instead of giving an error.

difosfor commented on docs/android-sdk.md:
```diff
@@ -0,0 +1,519 @@
+| `onVolumeChanged(float volume)`                                | Called when the volume of the player has changed.                                                                                          |
+| `onSourceChanged(String source)`                               | Called when the media source URL of the player has changed.                                                                                |
+
+### Registering Player Event Listener
```
This is a really conventional pattern in Android Java isn't it? Does it need to be extensively documented like this?

panchy replied:
Normally it should be used with setOnClickListener or addOnClickListener, but since the start it was registerListener so I left it as it is. Therefore an extra explanation is needed.

difosfor commented on docs/index.html:
```diff
@@ -64,5 +65,7 @@
     <script src="//unpkg.com/docsify-copy-code"></script>
 
     <script src="//unpkg.com/prismjs/components/prism-swift.min.js"></script>
+    <script src="//unpkg.com/prismjs/components/prism-java.min.js"></script>
```
Please order these prism scripts alphabetically.

difosfor commented on docs/android-sdk.md:
```diff
@@ -0,0 +1,519 @@
+
+Ex Machina Group Livery Android SDK.
+
+Documentation can be found at: [docs.liveryvideo.com/android-sdk](https://docs.liveryvideo.com/android-sdk).
```
I'd remove this; you're already there.

panchy commented on docs/android-sdk.md:
```diff
@@ -0,0 +1,519 @@
+
+Livery Android SDK is compatible with Android 5.0 (API level 21) or higher.
+
+### Download & Configure
```
Yes. That sounds better indeed. Should fix that.

difosfor commented:
Wait, why did you merge my LLS-653 branch into yours? That's tentative, spec and implementation are not finished yet, otherwise I would have committed or merged it to master already. Please reverse that commit (479e538) in this branch.

panchy commented:
> Wait, why did you merge my LLS-653 branch into yours? That's tentative, spec and implementation are not finished yet, otherwise I would have committed or merged it to master already. Please reverse that commit ([479e538](https://github.com/exmg/livery-docs/commit/479e53895767809e9e0d86ac4a1a5075d98a0dee)) in this branch.

I reverted it and rebased in another branch, fixed
